tests/foreman/api/test_rhci.py

```diff
@@ -0,0 +1,123 @@
+"""Unit tests for the ``activation_keys`` paths."""
+from ddt import data, ddt
+from fauxfactory import gen_string
+from nailgun import entities
+from requests.exceptions import HTTPError
+from robottelo.test import APITestCase
+# (too-many-public-methods) pylint:disable=R0904
```

Comment:
yeah, i left that as is from the previous file. i'll change the text in my upcoming version.